packages/cardpay-sdk/sdk/reward-pool/base.ts

```diff
@@ -267,6 +268,26 @@ export default class RewardPool {
     return this.addTokenSymbol(aggregateBalance(tokenBalances));
   }
 
+  async rewardTokenBalancesWithoutDust(
+    address: string,
+    safeAddress: string,
+    rewardProgramId?: string
+  ): Promise<WithSymbol<RewardTokenBalance>[]> {
+    const unclaimedValidProofs = (await this.getProofs(address, safeAddress, rewardProgramId, undefined, false))
+      .filter((o) => o.isValid)
+      .filter((o) => {
+        return o.gasEstimate.amount.lt(new BN(o.amount));
+      });
```

Comment:
I've noticed there are lot's of isValid and  gasEstimates filtering, maybe we can a function that returns valid proofs with/without crypto dust.

Reply:
I do like the suggestion. I would prefer to separate them out like this rather than have a single method

```
  async getUnclaimedValidProofs(
    address: string,
    rewardProgramId?: string,
    tokenAddress?: string
  ): Promise<WithSymbol<Proof>[]> {
  
  async getUnclaimedValidProofsWithoutDust(
    address: string,
    safeAddress: string,
    rewardProgramId?: string,
    tokenAddress?: string
  ): Promise<WithSymbol<ClaimableProof>[]> {
```